rnd/nextgenautogpt/server/server.py

```diff
@@ -0,0 +1,17 @@
+import multiprocessing as mp
+import time
+from typing import Any
+
+
+def run_server(server_to_manager: mp.Queue, manager_to_server: mp.Queue) -> None:
```

Comment:
Aren't we going to use fast api for this ?

Reply:
Yes we are